code/game/area/Space Station 13 areas.dm

```diff
@@ -2142,7 +2142,7 @@ proc/process_adminbus_teleport_locs()
 
 /area/awaymission/leviathan
 	name = "Leviathan"
-		
+
```

Comment:
@DeityLink if you use any sort of non shit code editor they automatically trim whitespace like that.